src/utils/lockup.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
do we still need this file given there is https://github.com/near/core-contracts/blob/master/scripts/deploy_lockup.sh ?